airflow/www/extensions/init_appbuilder.py

```diff
@@ -531,12 +525,7 @@ def add_view_no_menu(self, baseview, endpoint=None, static_folder=None):
         return baseview
 
     def add_api(self, baseview):
-        """
-            Add a BaseApi class or child to AppBuilder
-        :param baseview: A BaseApi type class
-        :return: The instantiated base view
-        """
-        return self.add_view_no_menu(baseview)
+        raise AirflowException("Airflow doesn't support the Flask AppBuilder REST API")
```

Comment:
Thoughts on doing this? Should help avoid new endpoints showing up automatically.

Reply:
Can we not remove this function entirely?

Reply:
🤦‍♂️ good point, thanks!